### PR TITLE
this is a safe evict , making descheduler more powerful

### PR DESCRIPTION
**Pull request by gj199575**
this is a safe evict , making descheduler more powerful

descheduler pr:https://github.com/kubernetes-sigs/descheduler/pull/1188

### CODE REVIEW

**Comment by william-wang** [comment]
@gj199575 Please log a issue firstly and describe your requirement and by the way please use english so that all the contributor can  get into the discussion.

**Comment by gj199575** [comment]
> @gj199575 Please log a issue firstly and describe your requirement and by the way please use english so that all the contributor can get into the discussion.

ok

File: pkg/scheduler/cache/cache.go [hunk 1]
```diff
@@ -161,7 +162,14 @@ func (db *DefaultBinder) Bind(kubeClient kubernetes.Interface, tasks []*scheduli
 	var errTasks []*schedulingapi.TaskInfo
 	for _, task := range tasks {
 		p := task.Pod
-		if err := kubeClient.CoreV1().Pods(p.Namespace).Bind(context.TODO(),
+		if p.Annotations["is-fake"] == "true" {
```
Comment (Thor-wl) [inline pkg/scheduler/cache/cache.go:165]: I'm sorry for not get the usage of `is-fake` annotation, can you add more description about this PR?

File: pkg/scheduler/cache/cache.go [hunk 2]
```diff
@@ -161,7 +162,14 @@ func (db *DefaultBinder) Bind(kubeClient kubernetes.Interface, tasks []*scheduli
...
+			db.CreateQueue()
+			p.Annotations["fake-pod"] = task.NodeName
+			p.Annotations["scheduling.k8s.io/group-name"] = "job1-e7f18111-1cec-11ea-b688-fa163ec79500"
```
Comment (Thor-wl) [inline pkg/scheduler/cache/cache.go:168]: Why this is a hard-code annotation with key 'group-name'?

**Comment by Thor-wl** [comment]
I'm sorry that I cannot merge this PR for there are serious problems both in issue description and code.

**Pull request closed without merging**